Makefile.include

```diff
@@ -327,7 +327,15 @@ $(info $(USEPKG:%=$(RIOTPKG)/%/Makefile.include))
 .PHONY: $(USEPKG:%=$(RIOTPKG)/%/Makefile.include)
 -include $(USEPKG:%=$(RIOTPKG)/%/Makefile.include)
 
-USEMODULE_INCLUDES_ = $(shell echo $(USEMODULE_INCLUDES) | tr ' ' '\n' | awk '!a[$$0]++' | tr '\n' ' ')
+# Deduplicate includes without sorting them
+# see https://stackoverflow.com/questions/16144115/makefile-remove-duplicate-words-without-sorting
+define uniq =
```

Comment:
Non-blocking, but maybe we should have a `makefiles/functions.mk` where we can centralize these kind of definitions. I think especially something like this can also be useful in other places.

Reply:
This `awk` uniq looks only used here but this could be better to refactor. There are other functions from the same kind.
I would keep it for another PR though.

Reply:
> I would keep it for another PR though.

Yes, that's why I said non-blocking ;-)